ee/api/ee_property_definition.py

```diff
@@ -18,8 +18,9 @@ class Meta:
             "updated_at",
             "updated_by",
             "query_usage_30_day",
+            "is_event_property",
```

Comment:
I think this could/should be a free feature.

The only thing we're tracking (in #7500) is which event has sent which properties, without any intelligence as to the content of the properties, nor how many times we have seen them, nor when. This is so we could split the "event properties" list into two --> properties that we have seen with this event & properties we have not seen with this event.

This feels like a basic feature, and I don't think it really falls under the "taxonomy" label. I can be convinced otherwise, but since we're anyway tracking this data on all instances, no matter if entreprise or not, it's also worth showing this data no matter the license.

Reply:
Yeah I'd be fine with this being a free feature, seems pretty basic